_includes/site-footer.html

```diff
@@ -5,13 +5,14 @@
     </span>
     <nav>
       <a href="{{ site.baseurl}}/nav-tips/" class="button button-nav-tips">Navigation tips</a>
+      <a href="{{ site.baseurl}}/print/" class="button button-print" target="_blank">Print this guide</a>
```

Comment:
Why the use of `_blank`?

Reply:
BC I didn't show a header/footer. I could add them back in and remove the `_blank`. I just wanted a way for people to get back to the guide and figured `_blank` solved it :)

Reply:
From the research I've seen -- which is far from definitive -- generally speaking you'd want to leave opening new tabs and navigating back in the hands of the users, reserving `_blank` for cases in which you've got a big workflow that could be lost, their place might be lost, or leaving and coming back could otherwise be destructive. That said, I think the research is a bit thin, and I definitely see your case for using it here.

Reply:
I harbor no strong preferences on `_blank`. I literally just talked myself through how I could design it better in response to your initial comment!